app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+
+    return counter
```

Comment:
The solution is correct, but it's better to use the `count()` function as it will simplify the code.